setup.py

```diff
@@ -20,9 +20,18 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #
 
+from setuptools import Command
 from setuptools import setup
-from DistUtilsExtra.command import (
-    build_i18n, clean_i18n, build_extra, build_icons)
+# for the time being do not use setuptools.command because:
+# * setuptools is to old in many distro (ex: Ubuntu 22.04)
+# * The Fedora package relies on wheel so it would not contains translation and
```

Comment:
Do you want me to look after the Fedora stuff? Pretty sure we can get rid of the wheel. Wheel's not ideal anyway ;) (It was just the easiest back when I first packaged this.)

Reply:
You read my mind. Yes !

Reply:
Just to be on the same page: The current version still works with the wheel but you want me to get rid of it in order to further improve the thing, correct?